src/android/ShortcutsPlugin.java

```diff
@@ -126,6 +126,14 @@ private void getIntent(CallbackContext callbackContext) throws JSONException  {
         Intent intent = this.cordova.getActivity().getIntent();
         PluginResult result = new PluginResult(PluginResult.Status.OK, buildIntent(intent));
         callbackContext.sendPluginResult(result);
+        Bundle data = intent.getExtras();
```

Comment:
For handle case
1. Application started from a static shortcut
2. Restart the application via window.location.href = '/'